lib/vdsm/tool/configurators/multipath.py

```diff
@@ -22,15 +22,13 @@
 import os
 import sys
 
-from vdsm.common import cmdutils
 from vdsm.common import commands
 from vdsm.storage import mpathconf
+from vdsm.storage.multipath import _MULTIPATHD
```

Comment:
We should not import private names. Make it public if needed. But why do we need the path to multiapthd in the configurator? It should really use a storage module for controlling the daemon.

We can move the code to introspect and control the multipathd daemon to vdsm.storage.multipathd. Or move the code using it to mpathconf?

Reply:
Moved it to storage.multipath, better solution indeed.